Cubical/Structures/MultiSet.agda

```diff
@@ -22,17 +22,17 @@ module _(A : Type ℓ)
 
  open Queues-on A Aset
 
- member-structure : Type ℓ → Type ℓ
- member-structure X = A → X → ℕ
+ count-structure : Type ℓ → Type ℓ
+ count-structure X = A → X → ℕ
 
- Member : Type (ℓ-suc ℓ)
- Member = TypeWithStr ℓ member-structure
+ Count : Type (ℓ-suc ℓ)
+ Count = TypeWithStr ℓ count-structure
 
- member-iso : StrIso member-structure ℓ
- member-iso (X , f) (Y , g) e = ∀ a x → f a x ≡ g a (e .fst x)
+ count-iso : StrIso count-structure ℓ
+ count-iso (X , f) (Y , g) e = ∀ a x → f a x ≡ g a (e .fst x)
 
- Member-is-SNS : SNS {ℓ} member-structure member-iso
- Member-is-SNS = SNS-≡→SNS-PathP member-iso ((λ _ _ → funExt₂Equiv))
+ Count-is-SNS : SNS {ℓ} count-structure count-iso
+ Count-is-SNS = SNS-≡→SNS-PathP count-iso ((λ _ _ → funExt₂Equiv))
```

Comment:
Aha, this is a place where we could just use https://github.com/agda/cubical/blob/master/Cubical/Foundations/Univalence.agda#L231 ?

Reply:
Or wait, maybe we need a more general version?

Reply:
It should be easier in this case actually. Maybe we can get rid of SNS-≡ completely in a separate PR?

Reply:
Yeah, that sounds good! I'll open an issue about it so that we don't forget